src/routes/v0/stream/users.rs

```diff
@@ -31,7 +34,10 @@ pub struct UserStreamQuery {
         ("skip" = Option<usize>, Query, description = "Skip N followers"),
         ("limit" = Option<usize>, Query, description = "Retrieve N followers"),
         ("source" = Option<UserStreamSource>, Query, description = "Source of users for the stream."),
-        ("depth" = Option<usize>, Query, description = "User trusted network depth, user following users distance. Numbers bigger than 4, will be ignored")
+        ("source_reach" = Option<StreamReach>, Query, description = "The target reach of the source. Supported in 'influencers' source."),
```

Comment:
To continue with the naming conventions, it should be `reach`

Reply:
this `reach` is related to the `source` so it's source's reach, hence the naming.

Reply:
The `reach` parameter name is concise, aligns with existing conventions, and avoids unnecessary redundancy. The swagger description already clarifies its relationship to the source. Another point, `source_reach` cannot be combined with the other `UserStreamSource` so better candidate, if you want to add a prefix, would be `influencer_reach` but I do not see neither...